registers.py

```diff
@@ -242,7 +242,13 @@ def c_values_array_name(self):
 
     def c_values_array_def(self):
         assert len(self.values)
-        arr_elem_def = (f'[{v.value}] = "{v.name}"' for v in self.values if v.value is not None)
+
+        # Remove whitespace from the names, so when they're displayed we can use
+        # only ' ' as a separator.
+        def remove_whitespace(s):
```

Comment:
`toCIdentifier()` can be used here, if you like.

Reply:
That works. Thanks.